### PR TITLE
Upgrade plugins & dependencies

### PR DESCRIPTION
**Pull request by wendigo**
Tested locally with Trino. Includes https://github.com/airlift/airbase/pull/392

### CODE REVIEW

**Comment by wendigo** [comment]
cc @Randgalt @martint @electrum

**Comment by wendigo** [comment]
PTAL @martint @electrum

File: airbase/pom.xml [hunk 1]
```diff
@@ -181,7 +181,7 @@
         <dep.logback.version>1.4.14</dep.logback.version>
         <dep.modernizer.version>2.7.0</dep.modernizer.version>
         <dep.opentelemetry.version>1.34.1</dep.opentelemetry.version>
-        <dep.opentelemetry-instrumentation.version>1.32.0</dep.opentelemetry-instrumentation.version>
+        <dep.opentelemetry-instrumentation.version>2.0.0</dep.opentelemetry-instrumentation.version>
```
Comment (martint) [inline airbase/pom.xml:184]: What's the impact of this update? It seems like a major change based on the version bump.

Comment (wendigo) [reply airbase/pom.xml:184]: The breaking change is:

```
The default OTLP protocol has been changed from grpc to http/protobuf in order to align with the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.28.0/specification/protocol/exporter.md#specify-protocol). You can switch to the grpc protocol using OTEL_EXPORTER_OTLP_PROTOCOL=grpc or -Dotel.exporter.otlp.protocol=grpc.
```

**Review by martint — APPROVED** [review]

**Comment by wendigo** [comment]
Rebased @martint

**Comment by wendigo** [comment]
@martint please merge and release new airbase version :)

**Pull request merged**